mod/notifications/views/default/notifications/subscriptions/forminternals.php

```diff
@@ -6,40 +6,30 @@
  */
 
```

Comment:
There already is an existing ticket to split this form... https://github.com/Elgg/Elgg/issues/4434